src/Simplex/Chat.hs

```diff
@@ -1004,8 +1063,18 @@ processChatCommand' vr = \case
                       withStore' (\db -> setContactDeleted db user ct)
                         `catchChatError` (toView . CRChatError (Just user))
                       pure $ map aConnId conns
+    CTLocal -> do
+      nf <- withStore $ \db -> getNoteFolder db user chatId
+      filesInfo <- withStore' $ \db -> getNoteFolderFileInfo db user nf
+      withChatLock "deleteChat local" . procCmd $ do
+        mapM_ (deleteFile user) filesInfo
+        -- functions below are called in separate transactions to prevent crashes on android
+        -- (possibly, race condition on integrity check?)
```

Comment:
we don't know anymore even for groups at the moment, maybe the issue is gone. for notes we haven't even UI yet :)
```suggestion
```

Reply:
not saying it's necessary to unite transactions, just remove comment